src/errorpage.cc

```diff
@@ -1523,3 +1523,12 @@ ErrorPage::ValidateStaticError(const int page_id, const SBuf &inputLocation)
     anErr.validate();
 }
 
+std::ostream &
+operator <<(std::ostream &os, const ErrorState *err)
+{
+    if (err)
+        os << errorTypeName(err->type);
```

Comment:
AFAICT, using errorTypeName() loses previously reported information when ErrorState::type is at or above `ERR_MAX` (i.e. when this is a so called "dynamically loaded" error). Fixed in 419899d.

N.B. ErrorState::type and ErrorState::page_id are pretty much the same, but fixing that bug is out of this PR scope.